lib/chef/resource/runit_service.rb

```diff
@@ -0,0 +1,693 @@
+
+      property :service_dir, String,
+        default: lazy { (node["runit"] && node["runit"]["service_dir"]) || "/etc/service" },
+        description: "The directory where services are symlinked to be supervised by runsvdir. By default, this will attempt to use the node['runit']['service_dir'] attribute, and falls back to /etc/service."
+
+      property :lsb_init_dir, String,
+        default: lazy { (node["runit"] && node["runit"]["lsb_init_dir"]) || "/etc/init.d" },
+        description: "The directory where an LSB-compliant init script interface will be created. By default, this will attempt to use the node['runit']['lsb_init_dir'] attribute, and falls back to /etc/init.d."
+
+      property :control, Array, default: [],
+               description: "An array of signals to customize control of the service, see runsv man page on how to use this. This requires that each template be created with the name sv-service_name-signal.erb."
+
+      property :env, Hash, default: {},
+               description: "A hash of environment variables with their values as content used in the service's env directory. Default is an empty hash. When this hash is non-empty, the contents of the runit service's env directory will be managed by Chef in order to conform to the declared state."
+
+      property :options, Hash, # deprecated: true should be set in the future
+        default: lazy { default_options }, coerce: proc { |r| default_options.merge(r) if r.respond_to?(:merge) },
+        description: "Options passed as variables to templates, for compatibility with legacy runit service definition. Default is an empty hash."
+
+      property :log, [TrueClass, FalseClass], default: true,
+               description: "Whether to start the service's logger with svlogd, requires a template sv-service_name-log-run.erb to configure the log's run script."
+
+      property :cookbook, String,
+        description: "A cookbook where templates are located instead of where the resource is used. Applies for all the templates in the enable action."
+
+      property :check, [TrueClass, FalseClass], default: false,
+               description: "Whether the service has a check script, requires a template sv-service_name-check.erb."
+
+      property :start_down, [TrueClass, FalseClass], default: false,
+               description: "Set the default state of the runit service to 'down' by creating <sv_dir>/down file. Defaults to false. Services using start_down will not be notified to restart when their run script is updated."
+
+      property :delete_downfile, [TrueClass, FalseClass], default: false,
+               description: "Delete previously created <sv_dir>/down file."
+
+      property :finish, [TrueClass, FalseClass], default: false,
+               description: "Whether the service has a finish script, requires a template sv-service_name-finish.erb."
```

Comment:
```suggestion
               description: "Determines whether the service has a finish script, which requires a template sv-service_name-finish.erb."
```

Reply:
Is the template supposed to be named "sv-service_name-finish.erb"? We might want to clarify that in this description and in the description in line 66.